scipy/signal/ltisys.py

```diff
@@ -375,6 +376,10 @@ class lti(object):
     of one of its subclasses: `StateSpace`, `TransferFunction` or
     `ZerosPolesGain`.
 
+    If (numerator, denominator) is passed in for `*system`, coefficients for
```

Comment:
This needs double backticks around both ``*system`` to render correctly in html/pdf docs (single backticks is for links to other objects/functions).